app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,13 @@
+class PagesController < ApplicationController
+  def index
+  end
+
+  def about
+  end
+
+  def blog
+  end
+
+  def contact
+  end
```

Comment:
Keep spaces between your class start and class end
```suggestion

  def index
  end

  def about
  end

  def blog
  end

  def contact
  end
  
```

Reply:
I've made this change